num_enum_derive/src/lib.rs

```diff
@@ -145,9 +294,123 @@ pub fn derive_into_primitive(input: TokenStream) -> TokenStream {
     })
 }
 
+/// Implements `From<Primitive>` for a `#[repr(Primitive)] enum`.
+///
+/// Turning a primitive into an enum with `from`.
+/// ----------------------------------------------
+///
+/// ```rust
+/// use num_enum::TryFromPrimitive;
```

Comment:
```suggestion
/// use num_enum::FromPrimitive;
```

Reply:
Fixed, alongside the other copy&paste issues in the code sample in question.